cores/arduino/wiring.h

```diff
@@ -24,6 +24,8 @@
 extern "C" {
 #endif
 
+#include "clock.h"
```

Comment:
Which compilation error does it fix ?  And why is this visible only with this new board ? 
Arduino.h is where wiring.h is included from and it first includes<chip.h> that should contain necessary defines.
Maybe the error should be corrected somewhere else ... depends on the actual error

Reply:
It fixes a compilation error in the library SPBTLE-RF because we call wiring.h and without the fix it doesn't find some function of clock.h.
I tried to use Arduino.h but I had more compilation errors due to missing prototypes.
Maybe there is another solution that could be implemented in the library and not in the core. I will continue to look for.

Reply:
Ok. I also think it is better to solve the library side and find the proper includes from the library

Reply:
This include is missing in the core and is required.
Wiring.h declare a static function which use GetCurrentMicro() define in clock.h.
Already see this issue in my test_usb branch on my forked repo.

Reply:
So I will leave it and we will use it as fix.

Reply:
OK  @fpistm is the boss :-)

Reply:
![Boss_Baby](https://upload.wikimedia.org/wikipedia/en/0/0e/The_Boss_Baby_poster.jpg)
But I can be wrong...